### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Freakshowdevil** [voroshkov/Solo-DIY-RF-Laptimer#4]
A quick question, Is it possible to have all the channels selectable via the app, is it a Arduino limitation or is the code not complete?
Nice work man :1st_place_medal:  ordering all part now and looking forward to the Multi-tracking Laptimer solution. How far have you come? How many RX5808 should I order :) and are you using Arduino Pro Mini for this one as well?

**Comment by voroshkov** [comment]
Yes, it's possible to make all bands/channels selectable. I intentionally limited them to simplify the app.

As for the multi-version: it will use the same hardware with minor modifications. My idea is to make the "Solo" devices attachable to each other to track several pilots during a race. Each racer may have his own fully functional device to track his solo flights, but when flying with friends, they just plug their laptimers like **A**->**B**->**C**.. -> **Bluetooth** and use the same hardware & software.
So far the Arduino code for the multi- version is complete and real Android app (not the AppInventor) is capable of communicating via bluetooth :)

**Comment by voroshkov** [comment]
So for the multi- solution you need solo- hardware multiplied by the number of devices you'd like to make :)

**Comment by Freakshowdevil** [comment]
That is brilliant, I am so excited about this 👍 
I'm ordering everything times 3.
Great if all band and channels are selectable, because some of my friends uses vtx that dont have raceband.

Super work, you're a king mate 👑

**Comment by voroshkov** [comment]
Thanks for the feedback!

**Comment by Freakshowdevil** [comment]
An idea for the start race. When you hitt start on the app the laptimer should wait 10 seconds before it starts tracking, the last 5 seconds of the wait timer the buzzer should beep each second and one long beep when race starts (start countdown).
And another thing is to set max lap count and long buzzer beep when max lap is reached for all quads. Then you don't have to count in your head when flying 😁

Just some ideas

**Comment by voroshkov** [comment]
Thanks for the ideas. However they are likely to be implemented in a next generation app :)

**Comment by Freakshowdevil** [comment]
Which pin number do you use to connect more RF-Laptimers together, so I can prepare this on my timers that I build.

**Comment by voroshkov** [comment]
So far my idea is to make it like this:
![imag2799](https://cloud.githubusercontent.com/assets/8790856/22454126/ca328910-e794-11e6-9d4d-65fa82a89802.jpg)
